.travis.yml

```diff
@@ -37,26 +37,57 @@ matrix:
      # Not even the latest WP version is compatible with PHPUnit 7. Manually install PHPUnit 6.
      script: curl -sSfL -o ~/.phpenv/versions/7.2/bin/phpunit https://phar.phpunit.de/phpunit-6.4.phar && phpunit
    - php: 7.3
-     env: WP_VERSION=trunk WC_VERSION=3.8.0-rc.1
+     env: WP_VERSION=latest WC_VERSION=3.8.1
      # Not even the latest WP version is compatible with PHPUnit 7. Manually install PHPUnit 6.
      script: curl -sSfL -o ~/.phpenv/versions/7.3/bin/phpunit https://phar.phpunit.de/phpunit-6.4.phar && phpunit
-   - php: 7.4snapshot
-     env: WP_VERSION=trunk WC_VERSION=3.8.0-rc.1
+   - php: 7.4
+     env: WP_VERSION=latest WC_VERSION=master
      # Not even the latest WP version is compatible with PHPUnit 7. Manually install PHPUnit 6.
-     script: curl -sSfL -o ~/.phpenv/versions/7.4snapshot/bin/phpunit https://phar.phpunit.de/phpunit-6.4.phar && phpunit
+     script: curl -sSfL -o ~/.phpenv/versions/7.4/bin/phpunit https://phar.phpunit.de/phpunit-6.4.phar && phpunit
 
      # Run the JS tests in a separate container
-   - language: node_js
-     before_script: ""
+  #  - language: node_js
+  #    before_script: ""
+   - name: "E2E tests"
+     php: 7.2
+     env: WP_VERSION=latest WP_MULTISITE=0 WC_VERSION=latest RUN_E2E=1
+     script:
+         - npm run test:e2e
+         - cat $HOME/wordpress/wp-content/debug.log
```

Comment:
Not sure what the `cat` does
but to address travis passing when npm test fails, maybe we can do something like
```
- npm run test:e2e; npm_exit_code=`$?`
 - cat $HOME/wordpress/wp-content/debug.log
- exit npm_exit_code
```
something like that to capture npm code and return after it.

Reply:
>something like that to capture npm code and return after it.

The way we do this in Core is by using `spawnSync()` instead of `spawn()` in `tests/bin/e2e-test-integration.js` to capture `jest` exit code and then pass it to `npm`:

https://github.com/woocommerce/woocommerce/blob/master/tests/bin/e2e-test-integration.js#L23-L41